things/models.py

```diff
@@ -7,7 +7,7 @@
 
 class Thing(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
-    titel = models.CharField(max_length=200)
+    title = models.CharField(max_length=200)
```

Comment:
I'm glad to see you fixed this bug. It looks like it might have been a little tricky to find